src/main/java/com/denizenscript/ddiscordbot/commands/DiscordChannelCommand.java

```diff
@@ -0,0 +1,149 @@
+package com.denizenscript.ddiscordbot.commands;
+
+import com.denizenscript.ddiscordbot.DenizenDiscordBot;
+import com.denizenscript.ddiscordbot.objects.DiscordChannelTag;
+import com.denizenscript.ddiscordbot.objects.DiscordGroupTag;
+import com.denizenscript.denizen.utilities.debugging.Debug;
+import com.denizenscript.denizencore.exceptions.InvalidArgumentsException;
+import com.denizenscript.denizencore.objects.Argument;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntry;
+import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+import com.denizenscript.denizencore.scripts.commands.Holdable;
+import net.dv8tion.jda.api.entities.Category;
+import net.dv8tion.jda.api.entities.TextChannel;
+import net.dv8tion.jda.api.requests.restaction.ChannelAction;
+import org.bukkit.Bukkit;
+
+public class DiscordChannelCommand extends AbstractCommand implements Holdable {
+
+    public DiscordChannelCommand() {
+        setName("discordchannel");
+        setSyntax("discordchannel [id:<id>] (group:<group_id>) (channel:<channel_id>) [create/delete] (name:<name>) (description:<description>) (category:<category_id>)");
+        setRequiredArguments(3, 6);
+    }
+    // <--[command]
+    // @Name discordchannel
+    // @Syntax discordchannel [id:<id>] (group:<group_id>) (channel:<channel_id>) [create/delete] (name:<name>) (description:<description>) (category:<category_id>)
+    // @Required 3
+    // @Maximum 6
+    // @Short Manages text channels on Discord.
+    // @Plugin dDiscordBot
+    // @Guide https://guide.denizenscript.com/guides/expanding/ddiscordbot.html
```

Comment:
I don't think the dDiscordBot guide page covers this at all, unless you're planning a followup guide PR? (imo redundant to do, better to just remove the guide link here)